app_guides/php/getting_started.adoc

```diff
@@ -0,0 +1,297 @@
+[[appguides-php-getting-started]]
+= Getting Started with PHP
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+=== Overview
+
+This tutorial walks through the creation of a PHP application titled `myapp`.
+Before going through the rest of the tutorial, be sure to have created an
+ifdef::openshift-online[]
+https://console.preview.openshift.com[{product-title} account]
+endif::[]
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+{product-title} account
+endif::[]
+and installed the `oc` command line tools.
+
+[[php-gs-step1]]
+== Step 1: The Sample Application and Git Repository
```

Comment:
These higher-level "Step 1" - "Step 3" titles don't serve much. Plus, what's here in step 1 is not a step, but context info. I think they could disappear.